roles/cinder/vars/main.yml

```diff
@@ -3,3 +3,4 @@ memcache_list: "{{ memcached_servers | join(', ') }}"
 public_ipaddr: "{{ hostvars[inventory_hostname][ 'ansible_' + primary_if ].ipv4.address }}"
 internal_ipaddr: "{{ hostvars[inventory_hostname][ 'ansible_' + private_if ].ipv4.address }}"
 cinder_osapi_volume_listen: "{{ internal_ipaddr }}"
+rabbit_hosts: "{% for node in groups['controller'] %}{{ node }}:5672{% if not loop.last %},{% endif %}{% endfor %}"
```

Comment:
Are you wanting the name of the node here or the IP address?  Something like `{{ hostvars[node][ 'ansible_' + primary_if ].ipv4.address }}:5672` maybe in place of `{{ node }}:5672`?